app/_how-tos/use-ai-rag-injector-plugin.md

```diff
@@ -0,0 +1,596 @@
+        data={'content': doc.page_content}
+    )
+    print(response.json())
+```
+
+{:.info}
+> You can replace `print(response.json())` with `print(response.text)` to view the raw HTTP response body as a plain string instead of a parsed JSON object. This is useful for debugging cases where:
+>
+> * The response isn't valid JSON (e.g., plain text error message or HTML).
+> * You want to inspect the exact response content without triggering a JSON parse error.
+>
+> Use `response.text` when troubleshooting unexpected server responses or plugin misconfigurations.
+
+
+Save the script as `inject_policy.py`, then run it in your terminal:
+
+```bash
+python ./inject_policy.py
+```
+
+This will output the number of chunks created and display the response from the injector endpoint for each chunk.
+
+```text
+Injecting 4 chunks...
+{"metadata":{"ingest_duration":1476,"embeddings_tokens_count":157,"chunk_id":"a1b2c3d4-e5f6-7890-ab12-34567890abcd"}}
+{"metadata":{"ingest_duration":1323,"embeddings_tokens_count":140,"chunk_id":"b2c3d4e5-f678-9012-bc34-567890abcdef"}}
+{"metadata":{"ingest_duration":1286,"embeddings_tokens_count":141,"chunk_id":"c3d4e5f6-7890-1234-cd56-7890abcdef12"}}
+{"metadata":{"ingest_duration":2892,"embeddings_tokens_count":168,"chunk_id":"d4e5f678-9012-3456-de78-90abcdef1234"}}
+```
+
+### Ingest content to the vector database
+
+Now, you can feed the split chunks into AI Gateway using the Kong Admin API.
+
+The following example shows how to ingest content to the vector database for building the knowledge base. The AI RAG Injector plugin uses the OpenAI `text-embedding-3-large` model to generate embeddings for the content and stores them in Redis.
+
+```bash
+curl localhost:8001/ai-rag-injector/{plugin_ID}/ingest_chunk \
+  -H "Content-Type: application/json" \
+  -d '{
+    "content": "<chunk>"
+  }'
+```
+{:.info}
+Replace the `plugin_ID` placeholder with your AI RAG Injector plugin ID.
+
+## Test RAG configuration
+
+### In-scope questions
+
+Use the following in-scope questions to verify that the AI responds accurately based on the approved compliance content and does not rely on external knowledge.
+
+{% navtabs "In scope" %}
+{% navtab "Basic questions" %}
+
+  Use simple user questions that map directly to travel policy clauses:
+
+  ```bash
+  curl --http1.1 localhost:8000/chat \
+    -H "Content-Type: application/json" \
+    -H "Authorization: Bearer $OPENAI_API_KEY" \
+    -d '{
+      "messages": [{"role": "user", "content": "Are alcoholic beverages reimbursable?"}]
+    }' | jq
+  ````
+
+  You can also ask this question:
+
+  ```bash
+  curl --http1.1 localhost:8000/chat \
+    -H "Content-Type: application/json" \
+    -H "Authorization: Bearer $OPENAI_API_KEY" \
+    -d '{
+      "messages": [{"role": "user", "content": "What documentation is required for travel reimbursement?"}]
+    }' | jq
+  ```
```

Comment:
````suggestion
  {% validation request-check %}
  url: /anything
  headers:
    - 'Content-Type: application/json'
    - 'Authorization: Bearer $DECK_OPENAI_API_KEY'
  body:
    messages:
      - role: user
        content: What documentation is required for travel reimbursement?
  {% endvalidation %}
````
Just be sure to double check my indentation and how this renders in the output :)